lib/Service/SetupService.php

```diff
@@ -91,7 +91,7 @@ public function startSetup(IUser $user, string $gatewayName, string $identifier)
 		try {
 			$gateway->send($user, $identifier, "$verificationNumber is your Nextcloud verification code.");
 		} catch (SmsTransmissionException $ex) {
-			throw new VerificationTransmissionException('could not send verification code');
+			throw new VerificationTransmissionException('could not send verification code', $ex->getCode(), $ex);
```

Comment:
On what place the app catch the 2th and 3th argument?

Reply:
The log-viewer will display a hierarchy of exception. This is handy for debugging. Otherwise you just see the top-level exception which provides zero information on what was going wrong.